components/CollectionCells/src/MDCCollectionViewCell.m

```diff
@@ -424,16 +424,32 @@ - (void)updateInterfaceForEditing {
 
 #pragma mark - Selecting
 
++ (NSBundle *)bundleWithName:(NSString *)bundleName {
+
+  NSBundle *baseBundle = [NSBundle bundleForClass:[self class]];
+  NSString *bundlePath = [baseBundle pathForResource:bundleName ofType:@"bundle"];
+  NSBundle *bundle = [NSBundle bundleWithPath:bundlePath];
+  NSAssert(bundle, @"Missing bundle %@.", bundleName);
+
+  return bundle;
+}
+
 - (void)setSelected:(BOOL)selected {
   [super setSelected:selected];
   if (selected) {
     _editingSelectorImageView.image =
-        [UIImage imageWithContentsOfFile:[MDCIcons pathFor_ic_check_circle]];
+                  [UIImage imageNamed:@"ic_check_circle"
+                             inBundle:[MDCCollectionViewCell
+                                          bundleWithName:@"MaterialIcons_ic_check_circle"]
+        compatibleWithTraitCollection:nil];
```

Comment:
As discussed offline, this should probably be moved in to the MDCIcon+<Name> generated by sync_icons.sh.